block/types.go

```diff
@@ -9,17 +9,22 @@ import (
 
 // TODO: move to types package
 
-type blockSource string
+type BlockSource uint64
```

Comment:
Starts with package name
should be Source

Reply:
moved to `types` package